<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="square_button_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>40</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="square_button_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>40</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="square_button_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>40</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="square_button_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>40</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="square_button_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>40</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="square_button_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>40</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="square_button_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>40</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="square_button_9">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>40</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="square_button_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>40</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Текст</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>260</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
